pac4j-oidc/src/main/java/org/pac4j/oidc/client/OidcClient.java

```diff
@@ -134,6 +136,11 @@
     /* max clock skew in seconds */
     private int maxClockSkew = DEFAULT_MAX_CLOCK_SKEW;
 
+    /* timeouts for token and userinfo requests */
+    private int connectTimeout = 0;
```

Comment:
What is the meaning of these values? Can we extract these default timeout values in the following class: https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/context/HttpConstants.java and reuse them here: https://github.com/pac4j/pac4j/blob/master/pac4j-oauth/src/main/java/org/pac4j/oauth/client/BaseOAuthClient.java#L58 ?